build/ci/phase-template.yml

```diff
@@ -34,9 +34,18 @@ phases:
       displayName: Download Benchmark Data
     - ${{ if eq(parameters.name, 'MacOS') }}:
       - script: brew update && brew install libomp mono-libgdiplus gettext && brew link gettext --force
-        displayName: Install runtime dependencies
+        displayName: Install runtime dependencies     
+    - script: $(_buildScript) -$(_configuration) -runtests -coverage=true
+      displayName: Run Tests with code coverage.
+      condition: and(eq(variables._phaseName, 'Windows_x64'), eq(variables._configuration, 'Debug'))
     - script: $(_buildScript) -$(_configuration) -runtests
-      displayName: Run Tests
+      displayName: Run Tests without code coverage.
+      condition: not(and(eq(variables._phaseName, 'Windows_x64'), eq(variables._configuration, 'Debug')))
+    - script: dotnet msbuild build/Codecov.proj /t:Codecov /p:UseCodecov=true
```

Comment:
I see that as part of your PR, if the `upload` step fails, it blocks my PR from merging. This is not intended, is it? If the codecov service fails for some reason, my PR is going to be blocked from merging?

Reply:
Failure to upload will not cause a failure with this configuration. See the following option:

https://github.com/codecov/codecov-bash/blob/4678d212cce2078bbaaf5027af0c0dafaad6a095/codecov#L138